models/intermediate/int_stripe__account_daily.sql

```diff
@@ -0,0 +1,103 @@
+with date_spine as (
+
+    select * 
+    from {{ ref('int_stripe__date_spine') }}
+
+), balance_transaction as (
+
+    select *,
+        case 
+            when type = 'payout' 
+            then {{ date_timezone('available_on') }}  
+            else {{ date_timezone('created_at') }}
+        end as date
+    from {{ ref('stripe__balance_transactions') }}
+
+), incomplete_charges as (
+
+    select *
+    from {{ ref('int_stripe__incomplete_charges') }}  
+
+), daily_account_balance_transactions as (
+
+    select
+        date_spine.date_day,
+        date_spine.account_id,
+        balance_transaction.source_relation,
```

Comment:
This field is showing as null for the days without transactions. To be consistent, let's use the date_spine source_relation field. Let me know if you have any concerns with this suggestion.
```suggestion
        date_spine.source_relation,
```

Reply:
Ooh good catch, updated